### DISCUSSION TITLE
If the attribute contenteditable="true" is used, it is impossible to insert pictures from the buffer.

### DISCUSSION THREAD

**Discussion by dmamel** [cefsharp/CefSharp#3885] [General]
Hi,
please help
If html content with contenteditable, then not work paste images from clipboard.
Maybe a paste process needs to be process handled somehow?

![image](https://user-images.githubusercontent.com/45320291/142201231-7579ec4a-a2eb-4c55-971c-320ecc46fb2c.png)

![image](https://user-images.githubusercontent.com/45320291/142200681-8dac95ec-8aec-4ac1-abd6-b17eb8186e27.png)

**Comment by amaitland** [comment]
Converting to discussion as bug report was not completed.

You'll need to test with the CEF Sample application (cefclient) to confirm the behaviour you are after is supported. 

https://github.com/cefsharp/CefSharp/blob/master/CONTRIBUTING.md#cefsharp-vs-chromium-embedded-frameworkcef
